stactask/__init__.py

```diff
@@ -1,12 +1,11 @@
+from contextlib import suppress
 from importlib.metadata import PackageNotFoundError, version
 
-try:
+with suppress(PackageNotFoundError):
     __version__ = version("stactask")
```

Comment:
With hatching writing the version file I would probably recommend using that file as the source of version info. Either way, if this package is not installed then you are going to get a situation where `__version__` is unbound.

This is my normal go-to for this:

```python
try:
    from .__version__ import __version__, __version_tuple__
except ImportError:
    __version__ = '0.0.0'
    __version_tuple__ = ('0', '0', '0')

__all__ = [
    '__version__',
    '__version_tuple__',
]
```